web/packages/teleterm/src/ui/DocumentGatewayApp/AppGateway.tsx

```diff
@@ -72,27 +73,13 @@ export function AppGateway(props: {
           />
         </Validation>
       </Flex>
-      {cliCommandPreview && (
-        <CliCommand
-          cliCommand={cliCommandPreview}
-          isLoading={props.changePortAttempt.status === 'processing'}
-          buttonText="Copy"
-          onButtonClick={props.copyCliCommandToClipboard}
-        />
-      )}
+      <Text>Access the app at:</Text>
+      <TextSelectCopy my={1} text={link} bash={false} />
```

Comment:
I wanted to use `TextSelectCopyMulti` component but it didn't look good due to its height. I'd have to modify the styling but I'm not sure if I want to (it is used in many places).